integreat_cms/cms/views/linkcheck/linkcheck_list_view.py

```diff
@@ -175,18 +175,20 @@ def post(self, request, *args, **kwargs):
                 }
                 # Replace the old urls with the new urls in the content
                 for translation in translations:
-                    new_translation = deepcopy(translation)
-                    # Replace link in translation
-                    logger.debug("Replacing links of %r", new_translation)
-                    new_translation.content = rewrite_links(
-                        new_translation.content,
-                        partial(self.replace_link, self.instance.url, new_url),
-                    )
-                    # Save translation with replaced content as new minor version
-                    new_translation.id = None
-                    new_translation.version += 1
-                    new_translation.minor_edit = True
-                    new_translation.save()
+                    # Replace if it is the latest version of the translation
+                    if translation == translation.latest_version:
+                        new_translation = deepcopy(translation)
+                        # Replace link in translation
+                        logger.debug("Replacing links of %r", new_translation)
+                        new_translation.content = rewrite_links(
+                            new_translation.content,
+                            partial(self.replace_link, self.instance.url, new_url),
+                        )
+                        # Save translation with replaced content as new minor version
+                        new_translation.id = None
+                        new_translation.version += 1
+                        new_translation.minor_edit = True
+                        new_translation.save()
```

Comment:
This check can be removed if we make sure the region_links only contains the latest versions anyway.


```suggestion
                    new_translation = deepcopy(translation)
                    # Replace link in translation
                    logger.debug("Replacing links of %r", new_translation)
                    new_translation.content = rewrite_links(
                        new_translation.content,
                        partial(self.replace_link, self.instance.url, new_url),
                    )
                    # Save translation with replaced content as new minor version
                    new_translation.id = None
                    new_translation.version += 1
                    new_translation.minor_edit = True
                    new_translation.save()
```